### DISCUSSION TITLE
replacing BRP069C81 / FDXM-F9 support on X35 / X50

### DISCUSSION THREAD

**Discussion by paulmhh** [revk/xESP32-Faikout#158] [Q&A] [answered]
Hello,

I saw that ADEA71A2VEB is supported (Sky Air, Ducted - Needs cable, uses X35A (power supply) and X50A (communication) ports)
that should be the replacement of BRP069C81 ... and then there should be hope for FDXM-F9 support.

Has anyone tried that yet?

Also the List of confirmed working AC units has the entry
"FTXM25F3V1B9 	Ducted 	Needs cable, uses X50 port"
but FTXM is wall mounted and not ducted. FDXM-F3 is ducted... typo?

thanks and great work!
Paul

**Comment by revk** [comment]
I'm not an expert on the model numbers so relying on people to update that wiki page as needed. I am working on supporting more models with CN_WIRED connection though.

**Comment by paulmhh** [comment]
do you happen to know if the daikin controllers (BRP069C81 in this case) tend to speak only one protocol? (because then the working of ADEA71A2VEB would more or less confirm the working of all BRP069C81 compatible devices)

**Reply by revk** [reply]
The controller I had tried S21 and X50 protocols, much like my Faikin does. I have a BRP069C41 I think.

**Reply by paulmhh** [reply]
if there is only one protocol (version) over X50 (pins)... then the faikin should support a lot of models, will let you know in 2-3 months

**Reply by paulmhh** [reply]
we will probably use FTXM and FDXM

**Reply by revk** [reply, marked as answer]
I have only heard of one protocol so far, but most users are on S21, so there may be variants. Given how consistent S21 has been I would hope X50A has only one protocol, at least at the basic level.

**Reply by eshroom** [reply]
I'm happy to give it a go, my units have BRP069C81 wifi controllers. @revk sent you a message on Twitter to try and get some of these as Amazon is a bit of a nightmare for me...

**Reply by revk** [reply]
Not seen a message

**Reply by eshroom** [reply]
Sorry, I meant a comment. On your last post about 300 sales (congratualtions btw!) as I can't send you PMs.

**Reply by revk** [reply]
Where are you - someone in the US is selling then on Tindie I think, if that helps.

**Reply by eshroom** [reply]
I'm in France, but AC units are at family home, where I am going for xmas. Don't think Amazon would get the here in time.

**Reply by revk** [reply]
I don't have any spare here, yet, maybe early next week.

**Reply by eshroom** [reply]
I remembered someone is actually flying from the UK so have ordered one from Amazon for them to bring. One question though, how does the Faikin control in HA compare to older B series wifi controllers that still have local access in HA, better/more stable, more limited/less stable, identical, same functionality?

I will report back on BRP069C81 though reading tells me it uses same protocol as BRP069A41 so no reason for it not to work.

**Reply by revk** [reply]
I think the HA stuff just works. No idea of the "older B series wifi", sorry.
Good luck.

**Reply by eshroom** [reply]
Thanks! What were you using for local http control in your last home? Not an older wifi controller?

**Reply by eshroom** [reply]
I can confirm working well as BRP069C81 replacement. Any chance of a private order for 6 more Faikins?

**Reply by revk** [reply]
If in the UK, we can ship direct, else simpler on Amazon. There is plenty of stock on Amazon now.

**Reply by eshroom** [reply]
Yes would be for UK. How do I go about ordering direct? Thanks.

**Reply by revk** [reply]
Email sales@aa.net.uk and FAO Adrian and I can sort. Ideally if you don't have an account, go to https://accounts.aa.net.uk/aal/new.cgi to make an account number, first.

**Reply by eshroom** [reply]
Have emailed you there. Many thanks!
